powdr_cli/src/main.rs

```diff
@@ -401,7 +384,12 @@ fn export_columns_to_csv<T: FieldElement>(
     // Write the column headers
     let headers = columns
         .iter()
-        .map(|(header, _)| header.clone())
+        .map(|(header, _)| {
+            header
+                .strip_prefix("Assembly.")
```

Comment:
I think we should remove any prefix that ends in `.` as long as the result is still unique.